CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.6.0] - 2019-09-25
+
+### Features
```

Comment:
```suggestion
### Added
```


    Added for new features.
    Changed for changes in existing functionality.
    Deprecated for soon-to-be removed features.
    Removed for now removed features.
    Fixed for any bug fixes.
    Security in case of vulnerabilities.

Reply:
Ah yeah, we have Added everywhere but 5.5.0 - I copied it from 5.5.0, hence the error.